apps/console/src/extensions/configs/authenticator.tsx

```diff
@@ -21,11 +21,12 @@ import { AuthenticatorConfig } from "./models";
 import { ChoreoButton } from "../components/shared/button/choreo-navigation-button";
 
 export const authenticatorConfig : AuthenticatorConfig = {
+    disableSMSOTPInSubOrgs: false,
```

Comment:
Let's send the cloud support PR as well.

Reply:
Done